### PR TITLE
2990 doc for 3rd party

### PR DESCRIPTION
**Pull request by martinhenningjensen**

### CODE REVIEW

File: doc/guide_3rd_party.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+var organizationId = "<organization-id>"; // Id of organization to act on behalf of
+
+using var httpClient = new HttpClient();
+using var request = new HttpRequestMessage(HttpMethod.Get, $"https://demo.energytrackandtrace.dk/wallet-api/wallets?organizationId={organizationId}");
+request.Headers.Authorization = new AuthenticationHeaderValue("bearer", token);
+request.Headers.Add("EO_API_VERSION", "20250101");
+var response = await httpClient.SendAsync(request);
+```
+
+Refer to ApenAPI spec for a description of available API endpoints.
```
Comment (ckr123) [inline doc/guide_3rd_party.md:115]: OpenAPI

Comment (martinhenningjensen) [reply doc/guide_3rd_party.md]: Corrected.

**Review by frontend-specialisten — COMMENTED** [review]
Consider using `markdownlint`, this is used by the frontend and other repo's across DataHub

**Comment by martinhenningjensen** [comment]
> Consider using `markdownlint`, this is used by the frontend and other repo's across DataHub

I have tried the VSCode plugin.

**Review by frontend-specialisten — APPROVED** [review]

**Pull request merged**